### ISSUE TITLE
Calling Object.getOwnPropertySymbols() on a Proxy throws an error

### ISSUE THREAD

**Issue by not-an-aardvark** [tvcutsem/harmony-reflect#71]
```js
require('harmony-reflect');
var myProxy = new Proxy({}, {});
Object.getOwnPropertySymbols(myProxy)
```

```
TypeError: Array.prototype.concat called on null or undefined
    at Function.getOwnPropertySymbols (native)
    at repl:1:8
    at REPLServer.defaultEval (repl.js:269:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:439:12)
    at emitOne (events.js:95:20)
    at REPLServer.emit (events.js:182:7)
    at REPLServer.Interface._onLine (readline.js:211:10)
    at REPLServer.Interface._line (readline.js:550:8)
```

This might be related to #57. Ideally, it would be nice if this didn't throw an error, even if the alternative is to just return an empty array every time. The current implementation causes an error to be thrown when calling certain lodash functions on Proxies, for example.

This behavior was tested on node 5.10.0.

**Comment by tvcutsem** [comment]
I just committed a fix implementing your suggestion: let `Object.getOwnPropertySymbols(proxy)` return `[]`. This shim still does not support symbols, but at least the patch stops code from crashing. The `getOwnPropertySymbols` method is now patched conditionally (if it exists).

Published on npm as harmony-reflect @ 1.4.5.

**Comment by not-an-aardvark** [comment]
Thanks!
